decidim-core/lib/decidim/core/api/user_group_type.rb

```diff
@@ -22,14 +22,16 @@ module Decidim
       resolve ->(obj, _args, _ctx) { UserGroupPresenter.new(obj).avatar_url }
     end
 
-    field :isVerified, !types.Boolean, "Whether the user group is verified or not", property: :verified?
```

Comment:
Also removed this field from the API since the `badge` field seemed enough. I could add it back with a better name (to account for officialization, like `hasBadge`), but I think think it's worth it just to make a single check in the client side more readable (`hasBadge === true` instead of `badge === ""`). Let me know anyways.